Userland/Libraries/LibGfx/TIFFGenerator.py

```diff
@@ -306,6 +306,24 @@ class Metadata;
             static_cast<double>(value.numerator) / value.denominator, value.numerator, value.denominator);
     }}
 }};
+
+template<>
+struct AK::Formatter<Gfx::TIFF::Value> : Formatter<FormatString> {{
+    ErrorOr<void> format(FormatBuilder& builder, Gfx::TIFF::Value const& value)
+    {{
+        String content;
+        value.visit(
+            [&](ByteBuffer const& buffer) {{
+                content = MUST(String::formatted("Buffer of size: {{}}"sv, buffer.size()));
+            }},
+            [&](auto const& other) {{
+                content = MUST(String::formatted("{{}}", other));
+            }}
+        );
+
+        return Formatter<FormatString>::format(builder, "{{}}"sv, content);
+    }}
+}};
```

Comment:
idle musing: It'd be kind of nice if there was a .h file we could put these generated snippets that don't have to be generated since they don't depend on anything. Maybe the generator could generate a OutputGenerated.h and then we could have Output.h that includes OutputGenerated.h but also contains code like this?

Reply:
It started to bother me too, I will move these snippets to another file next time I add one.